File: PHP/class/Aksesoris.php
<?php
require_once 'PetShop.php';

class Aksesoris extends PetShop {
    private $jenis;
    private $bahan;
    private $warna;

    public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna) {
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    public function get_jenis() {
        return $this->jenis;
    }

    public function get_bahan() {
        return $this->bahan;
    }

    public function get_warna() {
        return $this->warna;
    }
}
?>

File: Python/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
    def __init__(self):
        super().__init__()
        self.jenis = ""
        self.bahan = ""
        self.warna = ""

    def set_jenis(self, jenis):
        self.jenis = jenis

    def set_bahan(self, bahan):
        self.bahan = bahan

    def set_warna(self, warna):
        self.warna = warna

    def get_jenis(self):
        return self.jenis

    def get_bahan(self):
        return self.bahan

    def get_warna(self):
        return self.warna
